### PR TITLE
creating useful mockData & restoring unit test

### LINKED ISSUE #480: Restore skipped test suite(s)
**Description**
Upgrading @wagmi/core broke some of the test suites. They are currently being skipped to pass build and need to be debugged and restored. 

**Notes**
- balances.spec.ts is being skipped via `testPathIgnorePatterns: ['balances']`
- setup options tests are being skipped via `describe.skip` in ConnectWalletOptions.spec.ts
-  error message:
      FAIL  test/jest/__tests__/antelope/stores/balances.spec.ts
          ● Test suite failed to run
               ReferenceError: TextEncoder is not defined
- likely related to new wagmi version dependency on [viem](https://github.com/wagmi-dev/viem)

### PR DESCRIPTION
**Pull request by Viterbo**
# Fixes #480

Restoring disabled tests.

## Description

This PR put back in order all unit tests that were disabled while we developed the last urgent features. Now that we have done that we can restore the failing and disabled tests.

This PR also created a set of MockUps ready to use for future tests.

## Test scenarios
You can simply look at the test results on [this link](https://github.com/telosnetwork/telos-wallet/actions/runs/5824491662/job/15793724493?pr=522) or you can execute all by yourself locally with the following code:
`git fetch origin; git checkout 480-restore-skipped-test-suites; yarn; yarn test`

You should see all test passing.

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages
-   [x] I have included all english text to the translation file and/or created a new issue with the required translations for the currently supported languages
-   [x] I have tested for mobile functionality and responsiveness
-   [x] I have added appropriate test coverage

### CODE REVIEW

**Review by donnyquixotic — APPROVED** [review]

File: test/jest/__tests__/antelope/stores/balances.spec.ts [hunk 1]
```diff
@@ -1,204 +1,98 @@
...
+        test('__wagmiTokenTransferConfig should be {}', () => {
+            expect(JSON.stringify(store.__wagmiTokenTransferConfig)).toBe('{"current":null,"logged":null}');
+        });
     });
 
-    test('updateBalancesForAccount should update __balances', async () => {
-        const label = 'label';
-        const account = { account: 'address', authenticator: evmAuthenticatorMock };
-        await store.updateBalancesForAccount(label, account);
+    describe('Initializin the store', () => {
```
Comment (ezra-guzman) [inline test/jest/__tests__/antelope/stores/balances.spec.ts:35]:
```suggestion
    describe('Initializing the store', () => {
```

File: test/jest/__tests__/antelope/stores/balances.spec.ts [hunk 2]
```diff
@@ -1,204 +1,98 @@
...
-            label: [sysBalance, tokenBalance],
-        };
-        expect(JSON.stringify(store.__balances)).toBe(JSON.stringify(expected));
+            expect(useFeedbackStore).toHaveBeenCalled();
+            expect(FeedbackActions.setDebug).toHaveBeenCalled();
+        });
     });
 
+    describe('updateBalancesForAccount should update __balances', () => {
+        test('with the minimun system tokens when has zero balances for all tokens', async () => {
```
Comment (ezra-guzman) [inline test/jest/__tests__/antelope/stores/balances.spec.ts:48]:
```suggestion
        test('with the minimum system tokens when has zero balances for all tokens', async () => {
```

File: test/jest/utils/antelope/mockData.ts [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+const BALANCE_WRAPPED_TOKEN  = ethers.BigNumber.from('321'.concat('5'.repeat(18)));
+const BALANCE_STAKED_TOKEN   = ethers.BigNumber.from('456'.concat('7'.repeat(18)));
+const BALANCE_A_TOKEN        = ethers.BigNumber.from('789'.concat('8'.repeat(18)));
+const BALANCE_B_TOKEN        = ethers.BigNumber.from('987'.concat('9'.repeat(18)));
+
+// ADDRESS ---
+const ADDRESS_SYSTEM_TOKEN   = '0x5e734c1C5c3eE6aEeB7cD7b1c2d2bE7Ee0Ee0Ee0';
+const ADDRESS_WRAPPED_TOKEN  = '0xc8Fee563f0Ee0Ee0Ee0Ee0Ee0Ee0Ee0Ee0Ee0Ee0';
+const ADDRESS_STAKED_TOKEN   = '0x5e3c8Fee56C135Aecb733cd708cC31A5657aA30b';
+const ADDRESS_A_TOKEN        = '5ed7080xcC31A5657aA30b3c8Fee56C135Aecb733c';
```
Comment (ezra-guzman) [inline test/jest/utils/antelope/mockData.ts:17]: should this start with `0x`?

Comment (Viterbo) [reply test/jest/utils/antelope/mockData.ts:17]: :see_no_evil: :laughing:

**Review by ezra-guzman — APPROVED** [review]

**Pull request merged**